libnopegl/src/backends/gl/gpu_ctx_gl.c

```diff
@@ -540,13 +540,8 @@ static int gl_init(struct gpu_ctx *s)
 
     ngli_glstate_reset(gl, &s_priv->glstate);
 
-    const struct viewport vp = {NGLI_ARG_VEC4(config->viewport)};
-    if (ngli_viewport_is_valid(&vp)) {
-        ngli_gpu_ctx_set_viewport(s, &vp);
-    } else {
-        const struct viewport default_vp = {0, 0, config->width, config->height};
-        ngli_gpu_ctx_set_viewport(s, &default_vp);
-    }
```

Comment:
You still need to keep this chunk as-is as it would break the ngli_gpu_ctx_resize() API that accepts a null view port. This specific chunk will be removed in the following PR: https://github.com/NopeForge/nope.gl/pull/230